selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -152,12 +152,7 @@ void AdvancedNetworking::refresh() {
 }
 
 void AdvancedNetworking::toggleTethering(bool enable) {
-  if (enable) {
-    wifi->enableTethering();
-  } else {
-    wifi->disableTethering();
-  }
-  editPasswordButton->setEnabled(!enable);
+  wifi->toggleTethering(enable);
```

Comment:
toggleTethering implies you're just flipping the state. setTetheringEnabled would be a better name